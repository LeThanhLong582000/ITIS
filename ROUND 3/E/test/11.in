61
96325 24577
33745 91361
20001 68468
65251 18977
18849 69697
78413 24401
24081 62221
68055 43409
67881 39601
63275 43057
45626 16860
93217 36952
88423 42913
42441 17921
69779 46959
43361 34371
34287 27329
59936 80609
70403 84801
52488 58241
7181 92047
64545 4577
20889 69341
98021 37003
15009 45321
73649 90161
99651 98081
28321 59821
57743 16729
63825 97666
62763 61185
4377 37026
51017 63741
54845 33425
12001 29561
97650 16533
85249 22373
76306 83033
92209 68149
72893 79421
89601 21101
60151 91885
53433 38149
88133 66689
45329 28623
2010 31625
88801 93131
18801 35201
28866 8653
32081 80193
53953 29201
8729 1281
50577 15787
57121 62381
40001 55989
385 46725
12457 9351
34201 80001
87621 78517
32101 80417
48833 38597
