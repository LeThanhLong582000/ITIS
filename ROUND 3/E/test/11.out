-920810652
-617141648
45752808
-15626162
-1887212192
-720544036
-560358052
-71106174
1381900536
-710607162
1650144416
-508301884
1945416194
-464546008
-1889081108
-603549586
-1735874006
-1967257380
-1423254322
-1600445100
-2022209426
-1070718464
1048655220
417472946
2133448072
-1059476384
-1575429970
1437219724
1186995250
1739246886
727091638
-1746242410
-1962118588
808445004
288664312
-432831042
1493995780
2102473638
-300294972
-2071768408
-820164692
-1502973226
-784015460
1049196516
1193974074
989998718
1598035958
-994957392
-1761318370
925312464
144592144
-1441378888
-1123742106
-146222580
-1922766764
-148955612
-664235558
-503954248
-1146668440
1786947108
-867687772
